api/src/client/DynamicsBusinessIdsClient.test.ts

```diff
@@ -0,0 +1,74 @@
+import axios from "axios";
+import { BusinessIdClient } from "../domain/types";
+import { LogWriter, LogWriterType } from "../libs/logWriter";
+import { DynamicsBusinessIdsClient } from "./DynamicsBusinessIdsClient";
+
+jest.mock("axios");
+jest.mock("winston");
+const mockAxios = axios as jest.Mocked<typeof axios>;
+
+describe("DynamicsBusinessIdsClient", () => {
+  let client: BusinessIdClient;
+  let logger: LogWriterType;
+
+  let ORIGINAL_ORG_URL: string | undefined;
+  const ORG_URL = "www.test-org-url.com";
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    logger = LogWriter("NavigatorWebService", "ApiLogs", "us-test-1");
+
+    ORIGINAL_ORG_URL = process.env.DCA_DYNAMICS_ORG_URL;
+    process.env.DCA_DYNAMICS_ORG_URL = ORG_URL;
+
+    client = DynamicsBusinessIdsClient(logger, process.env.DCA_DYNAMICS_ORG_URL);
+  });
+
+  afterEach(() => {
+    process.env.DCA_DYNAMICS_ORG_URL = ORIGINAL_ORG_URL;
+  });
+
+  const mockAccessToken = "access-granted";
+  const mockNametoSearch = "Barcade";
+
+  const mockBusinessName1 = "Barcade123456";
+  const mockBusinessId1 = "123456";
+
+  const mockBusinessName2 = "Barcade789102";
+  const mockBusinessId2 = "789102";
+
+  it("makes a get request to dynamics api with the business ids query and auth token in header", async () => {
+    const businessIdMockResponse = {
+      value: [
+        {
+          name: mockBusinessName1,
+          accountid: mockBusinessId1,
+        },
+        {
+          name: mockBusinessName2,
+          accountid: mockBusinessId2,
+        },
+      ],
+    };
+
+    mockAxios.get.mockResolvedValue({ data: { ...businessIdMockResponse } });
+    expect(await client.getBusinessId(mockAccessToken, mockNametoSearch)).toEqual([
+      businessIdMockResponse.value[0].accountid,
+      businessIdMockResponse.value[1].accountid,
```

Comment:
[sand/dust] Since you're using variables so heavily, I think this might be more readable if you set up your expectation using the variable names instead of extracting them from the structure of the mock object.

Just something like:

```
 expect(await client.getBusinessId(mockAccessToken, mockNametoSearch)).toEqual([
      mockBusinessId1,
      mockBusinessId2
]
```